scenarios/16_Choosing_the_Best.cfg

```diff
@@ -236,6 +244,10 @@
             #po: HARD difficulty; supposed to be shorter and vaguer than on the two easier difficulties:
             {OBJECTIVE_NOTES (_"There is too little space to recall all 12 units at once. Move the units away to make room. The units will have “haste” (to borrow a term from a famous trading card game), so avoid ending your turn prematurely."+_" Important Note: do not UNDO the recall, units recalled are always counted even if you have undone the recall.")}
 #endif
+#ifdef NIGHTMARE
+            #po: NIGHTMARE difficulty:
+            {OBJECTIVE_NOTES (_"You can figure this out.")}
```

Comment:
No need for these kind of notes, just say nothing.

Reply:
Removed in [086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).